about.htm

```diff
@@ -0,0 +1,135 @@
+  <link rel="stylesheet" href="about.css">
+</head>
+<body>
+  <div class="background">
+
+  <header>
+    <nav>
+      <div class = "top-nav">
+        <ul class="nav-ul">
+            <li><a href="#" title="facebook"><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href="#" title="twitter"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+    </div>
+    <div class="bottom-nav">
+      <div class="icon">
+        <a href="index.html"><i class="fas fa-code fa-3x"></i>
+          <h1>LL Coding Together</h1>
+        </a>
+        <ul class="links">
+          <li class="link"><a href="index.html">Home</a></li>
+          <li class="link"><a href="">Program</a></li>
+          <li class="link"><a href="">Join</a></li>
+          <li class="link"><a href="">Sponsor</a></li>
+          <li class="link"><a href="">News</a></li>
+          <li class="link"><a id="border" href="">LL Coding</a></li>
+        </ul>
+      </div>
+    </div>
+
+      <i class="fas fa-bars fa-3x"></i>
+      <nav id="menu" class="invisible">
+        <ul class="menu-ul">
+            <li><i class="fal fa-times fa-3x cancel-icon"></i></li>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#About">Join</a></li>
+            <li><a href="#Contact">Sponsor</a></li>
+            <li><a href="#Contact">News</a></li>
+            <li><a href="#Contact">LL Coding</a></li>
+        </ul>
+    </nav>
+    </nav>
+  </header>
+
+  <main>
+    <section class="section-1">
+      <h2>"Hello World!"</h2>
+      <h1>LET'S LEARN CODING TOGETHER</h1>
+      <p>LL Coding Together is a non-profit organization that consists of an interactive learning web platform, an online community forum, chat rooms, online publications and local organizations that intend to make learning web development accessible to anyone.</p>
+      <article>
+        <p>Please contact us per Email for further questions about LL Coding Together.<span class="new bold">abebeb@gmail.com</span></p>
+      </article>
+
+    </section>
+  </main>
+</div>
+
+  <section class="section-2">
+    <h2>Let's Learn Coding Together Logo</h2>
+    <hr>
+    <p>The logo of LL Coding Together was decided through the logo competition that took place from June 8 to July 7.<span class="desktop">We have selected this logo because it got a lot of votes on our platform.</span></p>
+  </section>
+
+  <section class="section-3">
+    <i class="fas fa-code fa-5x"></i>
+    <div>
+      <h2>Let's Learn Coding Together</h2>
+      <p>Addis Ababa, Ethiopia, Africa</p>
+    </div>
+  </section>
+
+  <section class="section-4">
+    <h2>See the past LL Coding Together</h2>
+    <hr>
+    <p>Take a look at the last LL Coding Together in Asia and Europe.</p>
+    <div class="wrapper">
+
+      <a href="#">
+        <div class="past-event">
```

Comment:
- [ ] Please make the effect of this images to be orange instead of white in order to have a similar project to the design provided:

![image](https://user-images.githubusercontent.com/38522578/137225352-b373a99b-8b0e-4a3b-8941-1759a081fc59.png)

Reply:
This issue is well resolved! ✔️